Adyen.Test/ApplePayDeserializationTest.cs

```diff
@@ -1,23 +0,0 @@
-﻿using Adyen.Model.Checkout;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Newtonsoft.Json;
-
-namespace Adyen.Test
-{
-    [TestClass]
-    public class ApplePayDeserializationTest : BaseTest
-    {
-        [TestMethod]
-        public void Can_deserialize_applepay_paymentmethod_request()
-        {
-            var mockPath = GetMockFilePath("Mocks/checkout/paymentmethod-applepay-request.json");
```

Comment:
Adyen.Test/Mocks/checkout/paymentmethod-applepay-request.json is not used anymore right?
could you please remove it?

Reply:
yes done